### PR TITLE
added HGMD and other annotations

### PR DESCRIPTION
**Pull request by bw2**

### CODE REVIEW

File: xbrowse/core/constants.py [hunk 1]
```diff
@@ -394,12 +420,16 @@
     'name': 'intergenic_variant',
     'slug': 'intergenic_variant',
     'so': 'SO:0001628'},
-
 ]
 
+ANNOTATION_DEFINITIONS = CLINVAR_ANNOTATION_DEFINITIONS + VEP_ANNOTATION_DEFINITIONS
+ANNOTATION_DEFINITIONS_INTERNAL = ANNOTATION_DEFINITIONS + HGMD_ANNOTATION_DEFINITIONS + VEP_ANNOTATION_DEFINITIONS
+
```
Comment (hanars) [inline xbrowse/core/constants.py]: `ANNOTATION_DEFINITIONS` already  includes `VEP_ANNOTATION_DEFINITIONS` so you are actually adding it twice to `ANNOTATION_DEFINITIONS_INTERNAL`

File: xbrowse/core/constants.py [hunk 2]
```diff
@@ -507,23 +552,35 @@
 
         ]
     }
-
 ]
 
+
+ANNOTATION_GROUPS = CLINVAR_ANNOTATION_GROUPS + VEP_ANNOTATION_GROUPS
+ANNOTATION_GROUPS_INTERNAL = CLINVAR_ANNOTATION_GROUPS + HGMD_ANNOTATION_GROUPS + VEP_ANNOTATION_GROUPS
+
```
Comment (hanars) [inline xbrowse/core/constants.py]: same issue here

File: xbrowse/datastore/elasticsearch_datastore.py [hunk 3]
```diff
@@ -162,13 +171,18 @@ def get_elasticsearch_variants(self, project_id, family_id=None, variant_filter=
             # TODO add caching
             indiv_id = _encode_name(family_individual_ids[0])
 
-            mapping = self._es_client.indices.get_mapping(str(elasticsearch_index)+"*")
+
+            matching_indices = []
+            mapping = self._es_client.indices.get_mapping(str(elasticsearch_index)+"*,rare_genomes_project_wgs_rgp_*")
```
Comment (hanars) [inline xbrowse/datastore/elasticsearch_datastore.py]: why are we hardcoding RGP here?

File: xbrowse/core/constants.py (hunk 1 above)
Comment (bw2) [reply xbrowse/core/constants.py]: oops.. good call

File: xbrowse/core/constants.py (hunk 2 above)
Comment (bw2) [reply xbrowse/core/constants.py]: This one is alright I think

File: xbrowse/datastore/elasticsearch_datastore.py (hunk 3 above)
Comment (bw2) [reply xbrowse/datastore/elasticsearch_datastore.py]: that was a test for adding SVs, wasn't meant to be committed. Will take it out.

**Comment by bw2** [comment]
Fixed those things.

**Pull request merged**